### DISCUSSION TITLE
Failed to read list of home appliances

### DISCUSSION THREAD

**Discussion by fpaddict** [thoukydides/homebridge-homeconnect#109] [General]
I installed this plug in today and got it working. Rebooted and now I'm getting this error:
**[HomeConnect] Failed to read list of home appliances: TypeError: Cannot set property message of  which has only a getter**

Any ideas or could you point me to where to check for possible issues?

**Comment by thoukydides** [comment]
That sounds like it might be a bug.

Please raise an issue, and attach a full homebridge log with debug enabled (using the `-D` command line option to homebridge).

**Reply by fpaddict** [reply]
It looks like it resolved itself overnight. It seems to be working again today.

**Reply by thoukydides** [reply]
I found a bug that would account for the `TypeError: Cannot set property message of which has only a getter` error message, which is fixed in 0.26.2. However, that bug would have just changed the text of the error reported; it does not explain why the error occurred in the first place.
